lib/api/2.0/notification.js

```diff
@@ -30,7 +36,124 @@ var notificationProgressPost = controller(function(req, res){
     });
 });
 
+/**
+ * @api {post} /api/2.0/notification/alerts
+ * @apiDescription Receives Redfish alerts, refactors the message and publishs it into amqp
+ * @apiName notification post
+ * @apiGroup notification
+ */
+
+var notificationAlertsPost = controller({success: 201}, function(req,res) {
```

Comment:
Base our existing practice, the API controller function should be simple, the complex logic and implementation should go into the api service module. For notification, the module is https://github.com/tannoa2/on-http/blob/master/lib/services/notification-api-service.js

So please move your complex logic into `notification-api-service`